### ISSUE TITLE
Locale updates for brjs site docs

### ISSUE THREAD

**Issue by dchambers** [BladeRunnerJS/brjs#1187]
There have recently been 3 related changes that will need to be documented on the public site before we release:

  1. The locale no longer appears in the URL of single-locale apps.
  2. The `<@base.tag@/>` tag has been deprecated, but their is now an `<@unbundled-resources@/>` tag.
  3. Locale handling within BladeRunnerJS is now much more flexible.

Additionally, we will need some supporting information that can go in the release notes for people that are upgrading from an existing release.

**Comment by dchambers** [comment]
This issue is dependent on #1186.

**Comment by thanhc** [comment]
Closing this issue, as points 1 and 3 are done and reviewed, raised a new ticket for point two - this needs updating all docs to remove the base.tag from code snippets and mention about the unbundled-resources tag too.
